index.html

```diff
@@ -5,55 +5,108 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,900&subset=latin,cyrillic">
+    <link rel="stylesheet" href="style.css">
     <title>Gllacy</title>
 </head>
 
+
 <body>
-    <input type="radio">
-    <input type="radio">
-    <input type="radio">
-    <!--Доверстать радио кнопки, дать спили, найти спозоб сделать из них слайдер заднего фона. -->
+
+
+    <input class="visibility-hidden" type="radio" id="fon-1" name="toggle" checked>
+    <input class="visibility-hidden" type="radio" id="fon-2" name="toggle">
+    <input class="visibility-hidden" type="radio" id="fon-3" name="toggle">
+    <div class="slider-switches">
+        <div class="container">
+            <div class="wrapper">
+                <div class="slider-controls">
+                    <label class="slider-controls-item slider-controls-item-1" for="slider-1">
+                        <span class="visually-hidden">Первый</span>
+                    </label>
+                    <label class="slider-controls-item slider-controls-item-2" for="slider-2">
+                        <span class="visually-hidden">Второй</span>
+                    </label>
+                    <label class="slider-controls-item slider-controls-item-3" for="slider-3">
+                        <span class="visually-hidden">Третий</span>
+                    </label>
+                </div>
+            </div>
+        </div>
+    </div>
     <header>
-        <nav>
-            <a>
-                <img src="svg/logo.svg" alt="Логотип Gllacy" width="154" height="64">
-            </a>
-            <ul>
-                <li><a href="#">Каталог</a></li>
-                <li><a href="#">Доставка и оплата</a></li>
-                <li><a href="#">О Компании</a></li>
+        <a class="logo">
+            <img src="svg/logo.svg" alt="Логотип Gllacy" width="154" height="64">
+        </a>
+        <nav class="main-navigation">
```

Comment:
```suggestion
      <nav class="main-navigation">
        <a class="logo">
            <img src="svg/logo.svg" alt="Логотип Gllacy" width="154" height="64">
        </a>
```
логотип тоже элемент навигации